### ISSUE TITLE
Reraise netaddr.AddrFormatError as package exception

### ISSUE THREAD

**Issue by eliadl** [keiichishima/yacryptopan#5]
First, thanks for this repo. Much appreciated.

My usage is in a `try-except` clause, so when I pass an invalid address, `netaddr.AddrFormatError` is raised and I have to catch it.

Rather than force users like me to import this error from that other package, it makes more sense to have *this* package catch it and reraise it as an error of its own.

That way even if the inner logic changes, e.g. `netaddr` is replaced with something else, users can still expect the same error to be raised.

**Comment by keiichishima** [comment]
Thank you for your suggestion.

I was thinking your proposal for a while.  My concern was that introducing a new exception class in this module will break a backward compatibility.  Some users might be catching exceptions defined in the netaddr or the ipaddress module as you were doing.

But I finally decided to adopt your way.  At this moment, Python < 3.3 users are catching the netaddr  exception, and newer Python users are catching the ipaddress exception when parser failed.  Integrating these two exceptions may contribute ease of use of this module for future users.

I've modified my code and uploaded the latest version to the PyPi repository.
